racksdb/web/ui/src/views/DatacenterDetailsView.vue

```diff
@@ -7,16 +7,41 @@ SPDX-License-Identifier: GPL-3.0-or-later -->
 <script setup lang="ts">
 import { useHttp } from '@/plugins/http'
 import { useRacksDBAPI } from '@/composables/RacksDBAPI'
-import { ref, onMounted, watch } from 'vue'
-import SearchBar from '@/components/SearchBar.vue'
+import { ref, onMounted, watch, computed } from 'vue'
 import type { Ref } from 'vue'
 import type { Datacenter, DatacenterRoom } from '@/composables/RacksDBAPI'
 import BreadCrumbs from '@/components/BreadCrumbs.vue'
+import { BarsArrowDownIcon, BarsArrowUpIcon, ChevronRightIcon } from '@heroicons/vue/24/outline'
+import { useRouter } from 'vue-router'
 
+const router = useRouter()
 const http = useHttp()
 const racksDBAPI = useRacksDBAPI(http)
 const datacenters: Ref<Array<Datacenter>> = ref([])
 const datacenterDetails: Ref<Datacenter | undefined> = ref()
+const alphabeticalOrder = ref(true)
+const input = ref('')
+const filteredItem = computed(() =>
+  input.value === ''
+    ? datacenterDetails.value?.rooms
+    : datacenterDetails.value?.rooms.filter((item) => {
+        return item.name.toLocaleLowerCase().includes(input.value.toLocaleLowerCase())
+      })
+)
+
+function goToRoom(room: string) {
+  router.push({
+    name: 'datacenterroom',
+    params: { datacenterName: props.name, datacenterRoom: room }
+  })
+}
+
+function sortRooms() {
```

Comment:
This function does not sort the rooms, it inverts the sorting order. Please find a better name ;)

Reply:
I rename the function with a name more coherent, I called it invertRoomSort().